MNCNH obstructed labor Calculating Burden section (#1559)
ihmeuw/vivarium_research: docs/source/models/causes/maternal_disorders/gbd_2021_mncnh/obstructed_labor.rst

should accrue YLDs whether or not they die.

.. admonition:: Limitation

    The above strategy of computing average OL YLDs per
    case should correctly capture total YLDs for the acute sequela
    "obstructed_labor". However, **when
    we compute averted YLDs, the above calculation will not correctly
    count uncured or untreated fistula YLDs from the long-term sequelae 
    "rectovaginal_fistula" or "vesicovaginal_fistula"**, for two reasons:

    #. Fistula YLDs for a given age group will include not only OL or 
       uterine ruptures caused by current births, but by OL or 
       uterine ruptures caused by prior births. This means that we are 
       assigning extra YLDs to each current OL or uterine rupture case
       that are actually being accrued by other, nonpregnant people in
       the population who have lasting impacts of a previous birth and 
       have nothing to do with the OL or uterine rupture case we are modeling.

    #. If the modeled birth and uterine rupture case *does* lead to an
       uncured or untreated fistula, the total fistula YLDs will be spread 
       out over the simulant's remaining reproductive years, occurring in later

>        age groups, not entirely in the simulant's current age group.

Reviewer: ```suggestion
       age groups, not entirely in the simulant's current age group (when using the "prevalence YLD" approach currently employed by GBD).
```

>        Thus we will be "missing" a large portion of the YLDs caused by

Reviewer: ```suggestion
       Thus we will be missing a portion of the YLDs caused by
```

       the current birth events when we tally up YLDs for births in the
       simulant's current age group.

    Thus, if we avert a case of OL or uterine rupture, we will be simultaneously
    averting *extra* YLDs that we shouldn't be, because we are counting
    YLDs that don't actually belong to the simulant whose case was
    averted, as well as *missing* YLDs that should have been averted
    because we are only counting YLDs in the simulant's current age
    group, and not the YLDs that they would accrue in later years. Since
    births and hence incident cases of OL or uterine rupture `generally
    decrease with age <https://vizhub.healthdata.org/gbd-compare/#>`_, while cases of

>     uncured or untreated fistulas generally increase with age, we will 

Reviewer: ```suggestion
    uncured or untreated fistulas increase with age until age group 11, we will 
```
is age group 11 pretty young??

Author: Age group 11 is 30-34 years old

Reviewer 2: Maybe it would be better to say "until about age 34", which is easier to understand than "until age group 11".

    probably be systematically *undercounting* the YLDs that would be 
    averted by each averted case of OL, because for a OL case, the missed 
    YLDs for the simulant in question will on average be greater than 
    the extraneous YLDs from other simulants in the same age group.

>     It may be possible to develop a different strategy of counting YLDs

Reviewer: ```suggestion
    It may be possible to develop a different strategy of counting YLDs (such as switching to "incidence YLDs")
```

or uterine rupture in a given age group  will accrue the same
number of years lived with disability (YLDs). Specifically, the total
number of obstructed labor YLDs accrued by each affected simulant should
be the average number of YLDs per case of obstructed labor or uterine 
rupture in the simulant's age group, which is defined in the above data 
table as

.. math::

    \begin{align*}
    \text{ylds_per_case_c368}
        &= \frac{\text{OL YLDs}}{\text{OL cases}}\\
        &= \frac{\text{(OL YLDs) / person-time}}
            {\text{(OL cases) / person-time}}
        = \frac{\text{OL YLD rate}}{\text{OL incidence rate}}.
    \end{align*}

We are using the fact that  each simulant can get at most one case of
obstructed labor or uterine rupture during the simulation, so the average 
number of YLDs per affected simulant is the same as the average number of 
YLDs per case. Simulants with a case of obstructed labor or uterine rupture
should accrue YLDs whether or not they die.

.. admonition:: Limitation

    The above strategy of computing average OL YLDs per
    case should correctly capture total YLDs for the acute sequela

>     "obstructed_labor". However, **when

Reviewer 2: ```suggestion
    "obstructed labor, acute event". However, **when
```
Might as well use the exact sequela name

    we compute averted YLDs, the above calculation will not correctly
    count uncured or untreated fistula YLDs from the long-term sequelae 
    "rectovaginal_fistula" or "vesicovaginal_fistula"**, for two reasons:

    #. Fistula YLDs for a given age group will include not only OL or 
       uterine ruptures caused by current births, but by OL or 
       uterine ruptures caused by prior births. This means that we are 
       assigning extra YLDs to each current OL or uterine rupture case
       that are actually being accrued by other, nonpregnant people in
       the population who have lasting impacts of a previous birth and 
       have nothing to do with the OL or uterine rupture case we are modeling.

    #. If the modeled birth and uterine rupture case *does* lead to an
       uncured or untreated fistula, the total fistula YLDs will be spread 

>        out over the simulant's remaining reproductive years, occurring in later

Reviewer 2: ```suggestion
       out over the simulant's remaining lifetime, occurring in later
```
Fistulas are not specific to reproductive years like infertility is

       age groups, not entirely in the simulant's current age group (when using the "prevalence YLD" approach currently employed by GBD).
       Thus we will be missing a portion of the YLDs caused by
       the current birth events when we tally up YLDs for births in the
       simulant's current age group.

    Thus, if we avert a case of OL or uterine rupture, we will be simultaneously
    averting *extra* YLDs that we shouldn't be, because we are counting
    YLDs that don't actually belong to the simulant whose case was
    averted, as well as *missing* YLDs that should have been averted
    because we are only counting YLDs in the simulant's current age
    group, and not the YLDs that they would accrue in later years. Since
    births and hence incident cases of OL or uterine rupture `generally

>     decrease with age <https://vizhub.healthdata.org/gbd-compare/#>`_, while cases of

Reviewer 2: Here's a link showing incidence, since that's what you're talking about here: http://ihmeuw.org/6qzx

To generate such a link, after you select the parameters to generate the graph you want, you select "Share" in the upper right hand corner, and then click "Copy link".

Author: Ahh thank you! I meant to ask you about how you did that!

    uncured or untreated fistulas increase with age until age group 11 (and fistula YLDs can
    continue accruing all the way through the 95+ age group, unlike YLDs caused by sepsis
    or hemorrhage), we *might* be systematically *undercounting* the YLDs that would be 
    averted by each averted case of OL, because for a OL case, the missed 
    YLDs for the simulant in question will on average be greater than 

>     the extraneous YLDs from other simulants in the same age group. 

Reviewer 2: This may be fine as is, but we could leave it more vague and say something like: "It is unclear whether or not the overcounting and undercounting will balance out, so we may end up systematically overestimating or underestimating averted YLDs."
